### PR TITLE
Removes subject practioners page from teachcomputing

### PR DESCRIPTION
**Pull request by nyashamaphosa**
## Status

* Current Status: Ready for review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2361

## What's changed?

- *Removes the subject practioners page hosted on ghost pages from link or displaying teachcomputing pages and navigation*

### CODE REVIEW

File: spec/support/ghost/ghost_stubs.rb
Comment (dcorking) [inline spec/support/ghost/ghost_stubs.rb:29]: I think we still need this example: we should probably use another page

File: spec/requests/cms/cms_page_spec.rb
Comment (dcorking) [inline spec/requests/cms/cms_page_spec.rb:47]: I think we still need this example: unless we are doing away with support for nested pages. As above.

**Review by dcorking — CHANGES_REQUESTED** [review]
Good changes. Can you find out if we still need the nested CMS page feature, and depending on what you find, follow up my notes below?

File: spec/support/ghost/ghost_stubs.rb
Comment (nyashamaphosa) [reply spec/support/ghost/ghost_stubs.rb:29]: > Good changes. Can you find out if we still need the nested CMS page feature, and depending on what you find, follow up my notes below?

Message sent to Nikki the stackholder to get clarity on this

Comment (nyashamaphosa) [reply spec/support/ghost/ghost_stubs.rb:29]: > > Good changes. Can you find out if we still need the nested CMS page feature, and depending on what you find, follow up my notes below?
> 
> Message sent to Nikki the stackholder to get clarity on this

@dcorking I think we are good to take this off too
Screenshot 2023-05-22 at 13 40 13

**Review by dcorking — APPROVED** [review]
LGTM 🚂 

Ship it when you get approval from Nikki

**Pull request merged**
